typed/typed.go

```diff
@@ -81,18 +81,34 @@ func (tv TypedValue) Merge(pso TypedValue) (TypedValue, error) {
 	return merge(tv, pso, ruleKeepRHS)
 }
 
-// SymmetricDifference returns:
-//   * if both tv and rhs ("right hand side") give a value for a field:
-//     * if the value differs, then return rhs's value
-//     * otherwise, omit the field
-//   * if the field appears only in tv, return tv's value
-//   * if the field appears only in rhs, return rhs's value
+// Compare returns:
+//   * Any fields added by rhs ("right hand side") in 'added'.
+//   * Any fields changed by rhs in 'modified'.
```

Comment:
Can you clarify the documentation to say if `added` fields are included in `modified`?

Reply:
clarified